sprox/sa/provider.py

```diff
@@ -40,6 +40,9 @@
 from sprox.sa.widgetselector import SAWidgetSelector
 from sprox.sa.validatorselector import SAValidatorSelector
 from sprox._compat import string_type, zip_longest
+from sys import version_info
+
+PY2 = version_info[0] == 2
```

Comment:
You should just import this from `sprox._compat` ->https://github.com/TurboGears/sprox/blob/master/sprox/_compat.py#L9